streams/src/main/java/org/apache/kafka/streams/processor/internals/StateDirectory.java

```diff
@@ -62,34 +62,42 @@
         }
     }
 
+    /**
+     * Ensures that the state base directory as well as the application's sub-directory are created.
+     *
+     * @throws ProcessorStateException if the base state directory or application state directory does not exist and could not be created
+     */
     public StateDirectory(final String applicationId, final String stateDirConfig, final Time time) {
         this.time = time;
         final File baseDir = new File(stateDirConfig);
         if (!baseDir.exists() && !baseDir.mkdirs()) {
-            throw new ProcessorStateException(String.format("state directory [%s] doesn't exist and couldn't be created",
-                                                            stateDirConfig));
+            throw new ProcessorStateException(String.format("state directory [%s] doesn't exist and couldn't be created", stateDirConfig));
```

Comment:
any reason why you moved the `stateDirConfig` param onto the same line? It is past the 120 character mark - right?